app/cdap/text/text-en.yaml

```diff
@@ -3418,6 +3418,9 @@ features:
       gridTable:
         title: No rows left in this sample
         subtitle: Remove some recipe steps to view some data
+      connectionsList:
```

Comment:
make sure this is placed in features.WranglerNewUI

Reply:
this is related to NoRecordScreen feature

Reply:
Done